src/components/features/ClientFeedback/ClientFeedback.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './ClientFeedback.module.scss';
+import Feedback from '../../common/Feedback/Feedback';
+
+class ClientFeedback extends React.Component {
+  state = {
+    activePage: 0,
+  };
+
+  handlePageChange(newPage) {
+    this.setState({ activePage: newPage });
+  }
+
+  render() {
+    const { feedback } = this.props;
+    const { activePage } = this.state;
+
+    const currentFeedbackProps = feedback[activePage];
+    const pagesCount = Math.ceil(feedback.length);
+
+    const dots = [];
+    for (let i = 0; i < pagesCount; i++) {
+      dots.push(
+        <li>
+          <a
+            onClick={() => this.handlePageChange(i)}
+            className={i === activePage && styles.active}
+          >
+            page {i}
+          </a>
+        </li>
+      );
+    }
+
+    return (
+      <div className={styles.root}>
+        <div className='container'>
+          <div className={styles.panelBar}>
+            <div className='row no-gutters align-items-end'>
+              <div className={'col-auto ' + styles.heading}>
+                <h3>Client Feedback</h3>
+              </div>
+              <div className={'col '}></div>
+              <div className={'col-auto ' + styles.dots}>
+                <ul>{dots}</ul>
+              </div>
+            </div>
+          </div>
+          <div className={'row'}>
+            <Feedback {...currentFeedbackProps} />
```

Comment:
Spoko, i to w podstawowoym scenariuszy wystarczy, ale opinii może przzez długi czas nie być i feedback bęidze pustą tablicą, dlatego dodaj tu proszę warunek który sprawi, żę nie bęidzmey renderować tego komponenetu jeżeli feedback jest pustą tablicą